c/example04.c

```diff
@@ -0,0 +1,154 @@
+/* Example 04 :
+This example shows how to create and delete a tag for a specified resource.
+*/
+
+#include <string.h>
+#include <stdlib.h>
+#include <curl/curl.h>
+
+#define AK_SIZE 20
+#define SK_SIZE 40
+
+struct Tag {
+    char *_resources_id;
+    char *_key;
+    char *_value;
+    const char *_ak;
+    const char *_sk;
+    CURL *curl;
+};
+
+/* Function to create a tag */
+struct Tag *create_tag(struct Tag *tag)
+{
+    CURLcode res;
+
+    char json_data[BUFSIZ];
+    snprintf(json_data, sizeof(json_data), "{\"ResourceIds\": [\"%s\"], \"Tags\": [{\"Key\": \"%s\", \"Value\": \"%s\"}]}", tag->_resources_id, tag->_key, tag->_value);
+
+    /* Setting the URL for create */
+    curl_easy_setopt(tag->curl, CURLOPT_URL, "https://api.eu-west-2.outscale.com/api/v1/CreateTags");
+
+    /* Setting the AWS SigV4 and AK/SK */
+    curl_easy_setopt(tag->curl, CURLOPT_AWS_SIGV4, "osc");
```

Comment:
everything but CURLOPT_URL, and CURLOPT_POSTFIELDS should be initialize ouside create_tag and delete_tag